docs/Tutorials/OpenAPI.md

```diff
@@ -488,18 +673,226 @@ As JSON, this could look as follows:
+
+```powershell
+    New-PodeOAIntProperty -Name 'id'-Format Int64 -Example 1 -ReadOnly |
+        New-PodeOAStringProperty -Name 'username' -Example 'theUser' -Required |
+        New-PodeOAStringProperty -Name 'firstName' -Example 'John' |
+        New-PodeOAStringProperty -Name 'lastName' -Example 'James' |
+        New-PodeOAStringProperty -Name 'email' -Format email -Example 'john@email.com' |
+        New-PodeOAStringProperty -Name 'lastName' -Example 'James' |
+        New-PodeOAStringProperty -Name 'password' -Format Password -Example '12345' -Required |
+        New-PodeOAStringProperty -Name 'phone' -Example '12345' |
+        New-PodeOAIntProperty -Name 'userStatus'-Format int32 -Description 'User Status' -Example 1|
+        New-PodeOAObjectProperty  -Name 'User' -XmlName 'user'  |
+        Add-PodeOAComponentSchema
+
+    New-PodeOAStringProperty -Name 'street' -Example '437 Lytton' -Required |
+        New-PodeOAStringProperty -Name 'city' -Example 'Palo Alto' -Required |
+        New-PodeOAStringProperty -Name 'state' -Example 'CA' -Required |
+        New-PodeOAStringProperty -Name 'zip' -Example '94031' -Required |
+        New-PodeOAObjectProperty -Name 'Address' -XmlName 'address' -Description 'Shipping Address' |
+        Add-PodeOAComponentSchema
+
+    Merge-PodeOAProperty -Type AllOf -ObjectDefinitions 'Address','User'
+
+```
+
+As JSON, this could look as follows:
+
+```json
+{
+  "allOf": [
+    {
+      "$ref": "#/components/schemas/Address"
+    },
+    {
+      "$ref": "#/components/schemas/User"
+    }
+  ]
+}
+```
+
+
+## OpenApi Documentation pages
 
 If you're not using a custom OpenAPI viewer, then you can use one of the inbuilt ones with Pode - either Swagger and ReDoc, or both!
 
 For both you can customise the path to access the page on, but by default Swagger is at `/swagger` and ReDoc is at `/redoc`. If you've written your own custom OpenAPI definition then you can also set a custom path to fetch the definition.
 
-To enable either you can use the [`Enable-PodeOpenApiViewer`](../../Functions/OpenApi/Enable-PodeOpenApiViewer) function:
+To enable either you can use the [`Enable-PodeOAViewer`](../../Functions/OpenApi/Enable-PodeOAViewer) function:
 
 ```powershell
 # for swagger at "/docs/swagger"
 Enable-PodeOpenApiViewer -Type Swagger -Path '/docs/swagger' -DarkMode
 
-# or ReDoc at the default "/redoc"
+# and ReDoc at the default "/redoc"
 Enable-PodeOpenApiViewer -Type ReDoc
+
+# and RapiDoc at "/docs/rapidoc"
+Enable-PodeOAViewer -Type RapiDoc -Path '/docs/rapidoc' -DarkMode
+
+# and StopLight at "/docs/stoplight"
+Enable-PodeOAViewer -Type StopLight -Path '/docs/stoplight'
+
+# and Explorer at "/docs/explorer"
+Enable-PodeOAViewer -Type Explorer -Path '/docs/explorer'
+
+# and RapiPdf at "/docs/rapipdf"
+Enable-PodeOAViewer -Type RapiPdf -Path '/docs/rapipdf'
+
+# plus a bookmark page with the link to all documentation
+Enable-PodeOAViewer -Type Bookmarks -Path '/docs'
```

Comment:
Should this now not be `-Bookmarks` instead?

Probably worth having a separate section for the Bookmarks and Editor special viewers.

Reply:
yes but I forgot to change the documentation here :(

Reply:
I'm still seeing `-Type Bookmarks`